crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -303,55 +303,56 @@ pub enum MoveFunctionArgType {
 }
 
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
-pub struct TransactionEffectsResponse {
+pub struct SuiTransactionEffectsResponse {
```

Comment:
can we rename this to TransactionResponse/SuiTransactionResponse? Now effects is the only response type, we don't really need it in the name.

Reply:
Done!